demo/JENKINS_HOME/jenkins.install.UpgradeWizard.state

```diff
@@ -0,0 +1 @@
+2.0
```

Comment:
Otherwise we get an annoying banner at the top of the screen.

@kzantow note that the base Jenkins image does not set this, so you are prompted to install plugins. But in this case we already have all the relevant plugins preinstalled in the image. Perhaps the `README.md` for the image should be made to note that as of 2.0, when you are creating a derived image with specific configuration, you will generally want to add this file.